### DISCUSSION TITLE
Ant Media Server Customization

### DISCUSSION THREAD

**Discussion by hyalicom** [ant-media/Ant-Media-Server#5256] [Freelancing]
Hi, I have a quick Project, Where i need your help!

Installing AntMedia on Digital Ocean Server.
Streaming my RTMS link with different Audio (I want to Live Stream this Video, But due to Copy Right Issues i am not allowed to use the music in the Video Background, so i am looking for a way to replace the Video Music with my Own Music, which will be played in the Background in Loop! 

Anyone Interested who can help me in this, Please let me know! 

Thanks
Ali

**Comment by yashtandon113** [comment]
Hi @hyalicom
You can use Filter Plugin for this purpose as recommended to you by my colleague on below GitHub discussions.
https://github.com/orgs/ant-media/discussions/5255
